presto-main/src/test/java/io/prestosql/operator/TestTableFinishOperator.java

```diff
@@ -151,8 +151,8 @@ public void testStatisticsAggregation()
         assertBlockEquals(BIGINT, getOnlyElement(tableFinisher.getComputedStatistics()).getColumnStatistics().get(statisticMetadata), expectedStatisticsBlock);
 
         TableFinishInfo tableFinishInfo = operator.getInfo();
-        assertThat(tableFinishInfo.getStatisticsWallTime().getValue(NANOSECONDS)).as("statisticsWallTime nanoseconds").isGreaterThan(0);
-        assertThat(tableFinishInfo.getStatisticsCpuTime().getValue(NANOSECONDS)).as("statisticsCpuTime nanoseconds").isGreaterThan(0);
+        assertThat(tableFinishInfo.getStatisticsWallTime().getValue(NANOSECONDS)).as("statisticsWallTime nanoseconds").isGreaterThanOrEqualTo(0);
```

Comment:
Is that assertion still useful then? It probably mean to assert that the value gets updated

Reply:
I was hesitant whether to remove it or not. I kept it mostly as a pointer to impossibility to test this properly today, so that someon doesn't re-add it in the future without thinking.

thoughts?

Reply:
For DF we removed assertions and added comment in code that points to issue on github, see `io.prestosql.plugin.hive.TestHiveDynamicPartitionPruning#testJoinWithSelectiveBuildSide`